lib/import-resolver.js

```diff
@@ -70,17 +72,20 @@ ImportResolver.prototype.getFile = function (filename) {
 ImportResolver.prototype.normalizeImport = function (str) {
     if (regex.test(str)) {
         str = str.replace(regex, function (m, capture) {
-            var p,
-                pathToImport,
-                aliasKeys = Object.keys(this.aliases),
-                index = aliasKeys && aliasKeys.length ? aliasKeys.indexOf(capture) : -1;
-            if (index !== -1) {
-                pathToImport = this.aliases[aliasKeys[index]];
-            } else {
-                pathToImport = capture;
+            if(this.ignorePaths.indexOf(capture) === -1 && (!this.ignorePathPattern || !this.ignorePathPattern.test(capture))) {
```

Comment:
What do you think about inverting the base case here?

```js
var isIgnored = this.ignorePaths.indexOf(capture) !== -1;
var isPatternIgnored = this.ignorePathPattern && this.ignorePathPattern.test(capture)
if (isIgnored || isPatternIgnored) {
    return m;
}
// ...
```

To the reader, adding another nesting level makes this method seem more complicated than it actually is with these changes.